src/style.css

```diff
@@ -1,3 +1,75 @@
-body {
+:root {
+  --blue-color: #00acdc;
+}
+
+* {
   margin: 0;
+  padding: 0;
+  background-color: #fff;
+  font-weight: 500;
+  font-size: 12px;
```

Comment:
dont need to add it to all your elements,its impacts performance better option to add its in body